src/client/dfuse/ops/read.c

```diff
@@ -642,6 +642,6 @@ dfuse_pre_read(struct dfuse_info *dfuse_info, struct dfuse_obj_hdl *oh)
 		d_slab_release(eqt->de_pre_read_slab, ev);
 		active->readahead->dra_ev = NULL;
 	}
-	active_ie_decref(dfuse_info, oh->doh_ie);
+	active_ie_decref(dfuse_info, ie);
```

Comment:
This code holds a reference on the inode being "active" which itself holds a reference on the inode so it should be safe to access `ie` here.